src/NRediSearch/Client.cs

```diff
@@ -30,23 +31,21 @@ public enum IndexOptions
             /// <summary>
             /// The default indexing options - use term offsets and keep fields flags
             /// </summary>
-            Default = UseTermOffsets | KeepFieldFlags,
+            Default = UseTermOffsets | KeepFieldFlags | KeepTermFrequencies,
             /// <summary>
             /// If set, we keep an index of the top entries per term, allowing extremely fast single word queries
             /// regardless of index size, at the cost of more memory
             /// </summary>
-            UseScoreIndexes = 4,
-            /// <summary>
-            /// If set, we will disable the Stop-Words completely
-            /// </summary>
-            DisableStopWords = 8
+            KeepTermFrequencies = 4,
         }
 
-        public sealed class ConfiguredIndexOptions
```

Comment:
this break is ... problematic; I acknowledge that JRediSearch did this; I'm not sure we want to mirror their casual approach to compatibility :) needs thought, I think

Reply:
Yeah I think I might have gotten a little carried away with making NRediSearch "look" like JRediSearch. You think I should just revert my changes while taking into account the changes that need to be applied (missing and new index options)?

Reply:
Reverted.